src/Components/Application.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include"Controller.h"
+class Application {
```

Comment:
Please add a brief comment on what this class is, who calls it and who it calls.